doc/plotting/index.rst

```diff
@@ -363,3 +365,47 @@ NOTE: These functions works for only with matplotlib higher than 1.3.1.
                          :ref:`sphx_glr_auto_examples_01_plotting_plot_surf_stat_map.py`
 
 =====================   ===================================================================
+
+.. _interactive-surface-plotting:
+
+Surface plotting in a web browser
```

Comment:
I would call this function "Interactive surface plotting". The browser is more of an "implementation detail :).